cirq/google/api/v2/results.py

```diff
@@ -0,0 +1,174 @@
+from typing import (Dict, Iterable, Iterator, List, Optional, Set, Tuple, Union,
+                    cast)
+
+import numpy as np
+
+from cirq.api.google.v2 import result_pb2
+from cirq import circuits
+from cirq import devices
+from cirq import ops
+from cirq import schedules
+from cirq import study
+
+
+class Measurement:
+    """Info about a single measurement within a circuit or schedule."""
+
+    def __init__(self, key: str, qubits: List[devices.GridQubit], slot: int):
+        self.key = key
+        self.qubits = qubits
+        self.slot = slot
```

Comment:
suggest slot -> moment_index

Reply:
This is meant to be a more abstract notion of the location of the measurement in the program; it's moment index for circuits, but actual time for schedules. I've added a docstring to clarify. This is something we use internally when scheduling on hardware because we group together different measure ops that occupy the same "slot".